requirements.txt

```diff
@@ -40,6 +40,7 @@ opencv-python
 Pillow>=8.3.2
 matplotlib>=3.4,<3.5.3
 scipy
+nibabel
```

Comment:
This is required only for 110. I'd prefer to install it inside 110 instead of adding a new requirement for all.

Reply:
moved